src/software/thunderscope/replay/replay_file_debugging_script.py

```diff
@@ -52,7 +51,11 @@ def read_one_chunk(replay_file_name):
             #######################################
             # Do something with the protobuf here #
             #######################################
-            print(f"{line_num}: {float(timestamp)}: {protobuf_type} - {proto=}")
+            print(
+                "{}: {}: {} - {=}".format(
+                    line_num, float(timestamp), protobuf_type, proto
+                )
+            )
 
             line = replay_file.readline()
```

Comment:
There's still an extra `readline` here

Reply:
oops, that was code that I forgot to delete.